chapters/5-processing.tex

```diff
@@ -828,18 +828,28 @@ \subsection{Documenting data cleaning}
 so other researchers may use them for their analysis as well.
 
 \section{Looking ahead}
-Once you have corrected, cleaned, 
-and de-identified one or several data tables,
-and compiled the documentation to support it,
+This chapter introduced a process of exploring, understanding, and correcting
+the data that you obtained from the field or from partners.
+This process, called data cleaning and processing,
+creates your first research outputs using original data:
+an analysis-ready dataset.
+This dataset is a clear representation of reality (it is ``tidy''),
```

Comment:
this implied definition of tidy doesn't seem right to me

Reply:
```suggestion
This dataset is well-structured to describe your units of analysis (it is ``tidy''),
```